spot-client/src/spot-tv/ui/loaders/withCalendar.js

```diff
@@ -110,10 +109,7 @@ export class CalendarLoader extends AbstractLoader {
     _loadService() {
         calendarService.setConfig(
             this.props.calendarConfig,
-            [
-                ...this.props.meetingsDomainsWhitelist,
-                ...this.props.jwtDomains
```

Comment:
it is not clear to my why 'jwtDomains' merge goes away? I think it doesn't hurt to keep it

Reply:
jwt domains gets merged into whitelisted domains while setting default config.

Reply:
oh I've missed this part, nice